discovery/src/main/java/com/ibm/watson/discovery/v2/model/DocumentAccepted.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * (C) Copyright IBM Corp. 2019.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.discovery.v2.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.cloud.sdk.core.service.model.GenericModel;
+
+/**
+ * Information returned after an uploaded document is accepted.
+ */
+public class DocumentAccepted extends GenericModel {
+
+  /**
+   * Status of the document in the ingestion process. A status of `processing` is returned for documents that are
+   * ingested with a *version* date before `2019-01-01`. The `pending` status is returned for all others.
+   */
+  public interface Status {
+    /** processing. */
+    String PROCESSING = "processing";
+    /** pending. */
+    String PENDING = "pending";
+  }
+
+  @SerializedName("document_id")
+  private String documentId;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/watson-developer-cloud/java-sdk/pullRequest?prid=4552601)